Classes/Private/Manager/Submanagers/OCTSubmanagerCalls.m

```diff
@@ -140,13 +152,27 @@ - (BOOL)sendCallControl:(OCTToxAVCallControl)control toCall:(OCTCall *)call erro
 
         switch (control) {
             case OCTToxAVCallControlResume:
+
                 [self updateCall:call withStatus:OCTCallStatusActive];
+                if (! [self.audioEngine isAudioRunning:nil]) {
+                    [self.audioEngine startAudioFlow:nil];
+                }
+                self.audioEngine.friendNumber = friend.friendNumber;
+
                 break;
             case OCTToxAVCallControlCancel:
                 [self.timer stopTimer];
                 [self addMessageCall:call];
-                return [self.audioEngine stopAudioFlow:error];
+
+                if ((self.audioEngine.friendNumber == friend.friendNumber) &&
+                    ([self.audioEngine isAudioRunning:nil])) {
+                    return [self.audioEngine stopAudioFlow:error];
+                }
+
+                break;
             case OCTToxAVCallControlPause:
+                [self updateCall:call withStatus:OCTCallStatusPaused];
```

Comment:
I'm not sure what you mean here? I'm already doing that here?

        OCTFriend *friend = call.chat.friends.firstObject;

        if (! [self.toxAV sendCallControl:control toFriendNumber:friend.friendNumber error:error]) {
            return NO;
        }